unittesting/utils/output_panel.py

```diff
@@ -28,8 +30,13 @@ def __init__(
         settings.set("scroll_past_end", scroll_past_end)
         settings.set("syntax", syntax)
         self.closed = False
+        if capture_console:
+            self._stderr = sys.stderr.write
+        else:
+            self._stderr = lambda s: s
 
     def write(self, s):
+        self._stderr(s)
```

Comment:
I am not sure why we need this? I thought the stderr is already forwarded to the output panel if `capture_console` is true.

Reply:
I need it to be forwarded to the `sys.stderr` also.

Reply:
Check out this test: https://github.com/evandrocoan/DebugTools/blob/132219f3b271aff2036287e92bc6f98988bcf39a/tests/testing/main_unit_tests.py#L158-L175

Reply:
so you want to print the content of stderr without redirecting it?

Reply:
None. You need to redirect otherwise you cannot capture it and display it on the output panel. But I need you to send it to the standard `sys.stderr`. Doing such, you will not redirect, but just spy it and show on the output panel.

Reply:
Yes, printing it, while capturing it.

Reply:
You need to try to answer the question: How to test a logger of the `sys.stderr`?

Reply:
Currently, without this pull request, you cannot, because the UnitTesting package sends nothing to the default `sys.stderr`. So, the installed logger on `sys.stderr` cannot capture anything.

Reply:
And if it do not capture anything, you cannot assert anything as the test result.